addon/components/bs-form.js

```diff
@@ -82,29 +91,35 @@ import deprecateSubclassing from 'ember-bootstrap/utils/deprecate-subclassing';
   ### Submission state
 
   A `isSubmitting` property is yielded, which is `true` after submit has been triggered and before the Promise returned
-  by `onSubmit` is fulfilled. It could be used to disable form's submit button and showing a loading spinner for example:
+  by `onSubmit` is fulfilled. It could be used to show a loading spinner instead of the form while it's submitting for example:
 
   ```hbs
   <BsForm @onSubmit={{action "save"}} as |form|>
-    <BsButton @buttonType="submit" @disabled={{form.isSubmitting}}>
-      Save
-      {{#if form.isSubmitting}} {{fa-icon "spinner"}} {{/if}}
-    </BsButton>
+    {{#if form.isSubmitting}}
+      <FaIcon @icon="spinner" />
```

Comment:
Should we use a plain BS spinner example here, like this https://getbootstrap.com/docs/4.5/components/spinners/?

Juts in case people copy and paste this and wonder why it's not working ootb!?

Reply:
Done in 41ffa5bc93666ae856158c5bd072cca6d40c06e3. Also noticed that it might be good practice to not only show an icon but also have a text - at least for screen readers only (accessibility).